meddlr/data/transforms/2D_motion.py

```diff
@@ -0,0 +1,87 @@
+from typing import Optional, Tuple, Sequence, Union
+
+import math
+import numpy as np
+import torch
+import meddlr.ops as F
+from meddlr.transforms import RandomAffine, RandomTranslation, TransformList
+from meddlr.forward.mri import SenseModel
+
+from meddlr.utils.events import get_event_storage
+
+
+class MotionModel2D:
+    """A model that corrupts kspace inputs with motion.
+
+    Motion is a common artifact experienced during the MR imaging forward problem.
+    When a patient moves, the recorded (expected) location of the kspace sample is
+    different than the actual location where the kspace sample that was acquired.
+    This module is responsible for simulating different motion artifacts.
+    """
+
+    def __init__(self, nshots, angle, translate, trajectory, seed=None):
+        """
+        Args:
+            nshots (int) : The number of shots in the image. 
+                This should be equivalent to ceil(phase_encode_dim / 
+                echo_train_length). 
+            angle : The (min, max) angle for rotation. Values should be in 
+                degrees and should be >=-180, <=180. Use 'None' to 
```

Comment:
nit: use range notation instead `[-180, 180]`

Reply:
if this can be `None`, make sure it is listed as an optional variable: `angle: Optional[Tuple[int, int]]`. Given that it is optional, it should default to `None`